### DISCUSSION TITLE
[WP382 Discussion]:

### DISCUSSION THREAD

**Discussion by KestUrba** [wikipathways/wikipathways-help#52] [General]
### Discussion

There is the lack of a green pointer from **myc** (right upper box)  to **polyamines**.
It was detected for 26 kinds of cancer cells including pancreatic!

ref. Bachmann AS, Geerts D. **Polyamine synthesis as a target of MYC oncogenes**. J Biol Chem. 2018 Nov 30;293(48):18757-18769. doi: 10.1074/jbc.TM118.003336. Epub 2018 Nov 7. PMID: 30404920; PMCID: PMC6290138.

### Should this pathway be deleted?

- [ ] Yes, this pathway should be deleted

**Comment by khanspers** [comment]
This pathway described canonical MAPK signaling, and similarly to my comment in https://github.com/wikipathways/wikipathways-help/discussions/47, I think any specific connections between myc and polyamines as they relate to cancers belong in a separate model. There is no downside to have multiple versions of the same pathway, in fact this is often the case and better represents specific disease states, conditions etc.
